src/main/java/CustomerService/CustomerService.java

```diff
@@ -0,0 +1,42 @@
+package CustomerService;
+
+import javax.swing.*;
+
+public class CustomerService {
+    public static void main(String[] args) {
+    JFrame f=new JFrame("Frequently Asked Questions");
+    JTextField tf=new JTextField();
+    tf.setBounds(50,50, 2000,30);
+    JButton b=new JButton("How can I sell my product in your store");
```

Comment:
Make sure every question ends with a question mark to make it consistant with other question buttons.